### PR TITLE
CI: Add verbose option to avoid travis timeout for OSX install script 

### PR DESCRIPTION
**Pull request by sciunto**
## Description

Let's try this workaround for #4955

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

**Comment by sciunto** [comment]
Even worse. Let's try something else.

**Comment by sciunto** [comment]
nor working as expected...

**Comment by sciunto** [comment]
nice try @hmaarrfk but not working as well... :/

**Comment by sciunto** [comment]
looks like the verbose option helps to avoid the timeout.

**Comment by sciunto** [comment]
@scikit-image/core please squash before merging.

**Comment by sciunto** [comment]
@scikit-image/core If anyone has time to look at it. At least, it seems to solve the recent travis failure.

File: tools/travis/osx_install.sh [hunk 1]
```diff
@@ -7,8 +7,8 @@ brew tap homebrew/homebrew-cask
 brew cask install basictex
 
 export PATH="$PATH:/Library/TeX/texbin"
-sudo tlmgr --verify-repo=none update --self
-sudo tlmgr --verify-repo=none install ucs dvipng anyfontsize
+sudo tlmgr -v --verify-repo=none update --self
```
Comment (hmaarrfk) [inline tools/travis/osx_install.sh:10]:
```suggestion
# Add verbosity (-v) to avoid a timeout on travis
sudo tlmgr -v --verify-repo=none update --self
```

**Review by hmaarrfk — APPROVED** [review]

**Pull request merged**